ci/install_libseccomp.sh

```diff
@@ -0,0 +1,70 @@
+#!/bin/bash
+#
+# Copyright 2021 Sony Group Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+set -o errexit
+set -o nounset
+
+arch=$(uname -m)
+
+libseccomp_version="2.5.1"
+libseccomp_tarball="libseccomp-${libseccomp_version}.tar.gz"
+libseccomp_url="https://github.com/seccomp/libseccomp/releases/download/v${libseccomp_version}/${libseccomp_tarball}"
+libseccomp_install_dir="$1"
+cppflags="-O2"
+
+gperf_version="3.1"
+gperf_tarball="gperf-${gperf_version}.tar.gz"
+gperf_url="https://ftp.gnu.org/gnu/gperf/${gperf_tarball}"
+gperf_install_dir="$2"
```

Comment:
Conventionally, we store external dependency details in the "versions database" [here](https://github.com/kata-containers/kata-containers/blob/main/versions.yaml). Having a central database allows us to audit it (atleast in principle ;)

We then query the versions database using the `get_version()` function here:

- https://github.com/kata-containers/tests/blob/main/.ci/lib.sh#L150

For example:

- https://github.com/kata-containers/tests/blob/main/integration/cri-o/cri-o.sh#L54

Reply:
Thank you for the information and instruction. That makes sense.
I fixed it.